clustered/integration-test/src/test/java/org/ehcache/testing/StandardTimeouts.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ehcache.testing;
+
+import org.terracotta.utilities.test.matchers.Eventually;
+
+import java.time.Duration;
+
+public interface StandardTimeouts {
+
+  static Eventually.Timeout eventually() {
+    return Eventually.within(Duration.ofMinutes(1L));
```

Comment:
Should the timeout value be alterable by Java system property?

Reply:
I might argue no... this timeout is a trade-off of promptness of "true-positive" versus rate of 'false-positive' - if all tests are passing it should have no effect on test runtimes. For now the unification is to make it easier to bump in the future (if needed). I hope that we can just have a standard value everywhere and never need to tune it for certain environments.